# Paramters provided by the Ukart controller box

LeftnRight  mtrRPM;
LeftnRight  current;
LeftnRight  temp;
uint16_t  voltage;
uint16_t  yaw;
uint16_t  speedlinGoal;
uint16_t  speedAngGoal;
uint16_t  pitch;
uint16_t  roll;
uint32_t  odom;
